tests/test_model_builder.py

```diff
@@ -0,0 +1,294 @@
+"""Test model builder using variance of parfile input"""
+
+from glob import glob
+from os.path import basename, join
+from collections import defaultdict
+import pytest
+import io
+import astropy.units as u
+from pint.models.model_builder import (
+    ModelBuilder,
+    ConflictAliasError,
+    UnknownBinaryModel,
+    ComponentConflict,
+)
+from pint.models.timing_model import PhaseComponent, Component
+from pint.models.parameter import floatParameter
+from pint.utils import split_prefixed_name, PrefixError
+
+
+test_par1 = """
+PSR              B1855+09
+#RAJ             18:57:36.3932884         1  0.00002602730280675029
```

Comment:
Why leave these commented lines? Please make the par file as simple and to the point as possible.